src/components/BingoBoardCard.vue

```diff
@@ -72,16 +76,35 @@ setInterval(() => {
 }, 200);
 
 function start() {
-  show.value = true;
+  startBtn.value = false;
+  showBingo.value = true;
   startBingo.value = setInterval(() => {
-    bingo.value = ref(Math.floor(Math.random() * 99) + 1);
-    if (bingoNumbers.value.includes(bingo.value._value)) {
-      isBingo.value = true;
-    }
+    bingo.value = Math.floor(Math.random() * 99) + 1;
   }, 300);
 }

```

Comment:
add comments on function with what is the use of the function

Reply:
okay